gulp/tasks/scripts.js

```diff
@@ -42,14 +35,7 @@ function exampleScripts() {
       output: {
         filename: 'example.js'
       },
-      plugins: [
-        // Change warnings flag to true to view linter-style warnings at runtime.
-        new webpack.optimize.UglifyJsPlugin( {
-          compress: { warnings: true }
-        } )
-      ]
```

Comment:
I removed the minification step because IMHO it's premature optimization. This package isn't used on its own -- it's consumed by other projects -- so I think it's appropriate to leave minification to the consumer. Thoughts?

Reply:
One reason to keep it may be to keep the build process as close to other projects as possible, which will make it easier to update them all. Having recently updated webpack in CF, DM, and here it helps if the config looks and acts the same.

Reply:
Valid point. I think I'd prefer to remove it from most of our projects. It often makes development harder by obfuscating the transpiled code. For example, when I write JS for cfgov-refresh I find myself temporarily commenting out the uglify step to make debugging IE a lot easier. It seems like the sort of thing that should be in a deployment pipeline job or a separate gulp task or something.

Reply:
Ah yeah, we were having this discussion in CF a little while ago (b/c I crushed the unminified version 😛 ). The way I've been seeing to do this is set `NODE_ENV` and have a development and production setting https://stackoverflow.com/questions/25956937/how-to-build-minified-and-uncompressed-bundle-with-webpack

webpack has a flag for the environment too https://webpack.js.org/api/cli/#environment-options

In that setup each project has a set of environment settings and while some may omit one pathway, the pathways they share could be the same.

Reply:
I'm definitely on board with this. Do we have it saved somewhere so it doesn't get lost to the ether?